### PR TITLE
7.2 - HexMex - Jamie Simper

### PR DESCRIPTION
**Pull request by jasimper**

### CODE REVIEW

**Comment by rpearce** [comment]
What episode is this Spok shot from??

File: app.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+h3 {
+  display: inline-block;
+  width: 50%;
+  font-size: 1.5rem;
+  padding-left: 10px;
+}
+p {
+  display: inline-block;
+  width: 50%;
+  font-size: .7rem;
```
Comment (rpearce) [inline app.css]: I recommend prefixing this with a `0`. People often miss the `.` and are like `7 rem`??!?!?

**Comment by rpearce** [comment]
Nice work! I believe we refactored this in class

**Comment by jasimper** [comment]
I think that is Isis, who is actually a beautiful shapeshifter so this would be Assignment:Earth, which I believe is the season 2 finale.

**Pull request merged**